README.md

```diff
@@ -10,58 +10,35 @@ We developed J2V8 as a high performance engine for our multi-platform mobile too
 
 Building J2V8
 =============
-Building J2V8 requires building both the native parts and the Java library (.jar file). To build the native parts we first build node.js as a library and then statically link J2V8 to that. The Java parts are built with maven.
-
-Building on MacOS
------------------
-```
- sh ./build-node.sh
- sh ./buildJ2V8_macos.sh
- mvn clean verify
-```
-
-Building on Linux
------------------
-```
-export CCFLAGS="${CCFLAGS} -fPIC"
-export CXXFLAGS="${CXXFLAGS} -fPIC"
-export CPPFLAGS="${CPPFLAGS} -fPIC"
-#sh ./build-node.sh
-cp -r /data/jenkins/node .
-cd jni
-g++ -I../node -I../node/deps/v8 -I../node/deps/v8/include \
-    -I../node/src -I /data/jenkins/tools/hudson.model.JDK/jdk-7/include/ \
-    -I /data/jenkins/tools/hudson.model.JDK/jdk-7/include/linux  \
-    com_eclipsesource_v8_V8Impl.cpp -std=c++11 -fPIC -shared -o libj2v8_linux_x86_64.so \
-    -Wl,--whole-archive ../node/out/Release/libnode.a  -Wl,--no-whole-archive \
-    -Wl,--start-group \
-                      ../node/out/Release/libv8_libbase.a \
-                      ../node/out/Release/libv8_libplatform.a \
-                      ../node/out/Release/libv8_base.a \
-                      ../node/out/Release/libv8_nosnapshot.a \
-                      ../node/out/Release/libuv.a \
-                      ../node/out/Release/libopenssl.a \
-                      ../node/out/Release/libhttp_parser.a \
-                      ../node/out/Release/libgtest.a \
-                      ../node/out/Release/libzlib.a \
-                      ../node/out/Release/libcares.a \
-    -Wl,--end-group \
-    -lrt -D NODE_COMPATIBLE=1
-mvn clean verify
-```
-
-This will build J2V8 with node.js support. To disable this support, remove the `-D NODE_COMPATIBLE=1` option.
-
-Building for Android
------------------
-Building J2V8 for Android requires Docker.
-
-```
-./docker/build.sh
-./gradlew assembleRelease
-```
-
-This will build J2V8 as an AAR for API 19 minimum with node.js support.
+Building J2V8 requires building both the native parts and the Java library (.jar/.aar file). To build the native parts we first build node.js as a library and then statically link J2V8 to that. The Java parts are built with maven/gradle.
+
+J2V8 uses a cross-platform, cross-compiling build-system written in Python.
+
+Follow these steps to build J2V8 from source:
+
+1) clone the Node.js source code
+    - `python prepare_build.py`
```

Comment:
> was there any reason why you choose python over lets say bash scripts? I am just thinking out loud here that not everyone has python2 installed.

There are several points why I would not use bash scripts for more complex scenarios
- the composability of bash/shell relatively limited compared to python's (functional reuse, data reuse, ...)
- bash is only natively available on MacOS / Linux environments (python covers all platforms and also delivers platform independent APIs for OS functions, e.g. file access)

I targeted Python 2 primarily because you already need it if you want to [build Node.js](https://github.com/nodejs/node/blob/master/BUILDING.md#building-nodejs-on-supported-platforms) from source. But I will look into how much work it would take to make it compatible for Python 3 (but I don't think this will be a priority, since Python 2 is still very widely adopted)

About the error you get from running `prepare_build.py` ... this is mainly a utility script that is inteded to get you started from a fresh checkout of the J2V8 repository, for the case when there is already a `node` directory checked out this script should not try to override anything you already have there, but I will add a check for that and print some more helpful error message, thanks for pointing this out.